content/yaml/building-a-flutter-app.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Building a Flutter app
+description: Building a Flutter app with YAML.
+weight: 2
+---
+
+With `codemagic.yaml`, you can use Codemagic to build, test and publish Flutter apps. You can read more about how to use codemagic.yaml and see the structure of the file [here](../yaml/yaml).
```

Comment:
Similar to native app templates, you could explain that this page focuses on the script section as this is what changes depending on the framework used